web/src/components/atoms/DownloadButton/index.tsx

```diff
@@ -18,8 +18,15 @@ const DownloadButton: React.FC<DownloadButtonProps> = ({
   displayDefaultIcon,
   ...props
 }) => {
-  const handleDownload: MouseEventHandler<HTMLElement> | undefined = () => {
-    fileDownload(url, filename);
+  const handleDownload: MouseEventHandler<HTMLElement> | undefined = async () => {
```

Comment:
Please start using useCallback on basically all functions inside a component. A while ago we discussed with @rot1024 and the conclusion was basically "why not? if we do then we know the functions won't rerender needlessly".

Reply:
Done!